src/pages/students/forms/paymentDataForm/PaymentData.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  MainContainer,
+  FormContainer,
+  Label,
+  Input,
+  Button,
+  ButtonsContainer,
+  Select,
+  InputsContainer,
+  Text
+} from "./PaymentData.styles";
+import { Controller, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+import { paymentDataScheme } from "./PaymentData.scheme";
+import { ErrorMessage } from "../../../../components/error_message/ErrorMessage"
+import type { FormProp } from "../../create-student/FormProp.type";
+import { PaymentPlanName, PlanTypeNameArray } from "../../../../app/types/models/PaymentPlanName";
+import CurrencyInput from "../../../../utils/InputPrice/CurrencyInput"
+import { forwardRef, useImperativeHandle } from "react";
+
+export interface PaymentDataProps {
+  paymentPlanName: string;
+  extraClasses?: number | null | undefined;
+  classPrice?: number | null | undefined;
+  paymentDay?: number | undefined;
+}
+
+const PaymentData = forwardRef<FormProp<PaymentDataProps>, FormProp<PaymentDataProps>>((props, ref) => {
+
+  const { data, onNext } = props;
+
+  type FormData = yup.InferType<typeof paymentDataScheme>;
+
+  const DEFAULT_PAYMENT_DATA: PaymentDataProps = {
+    paymentPlanName: "",
+    extraClasses: null,
+    classPrice: undefined,
+    paymentDay: undefined
+  };
+
+  const studentRegistrationForm = data || DEFAULT_PAYMENT_DATA;
+
+  const {
+    register,
+    handleSubmit,
+    watch,
+    control,
+    formState: { errors },
+  } = useForm<FormData>({
+    resolver: yupResolver(paymentDataScheme) as any,
+    defaultValues: { ...studentRegistrationForm },
+  });
+
+  const PaymentPlanNameSelected = watch("paymentPlanName");
+
+  useImperativeHandle(ref, () => ({
+    submit: () =>
+      new Promise<boolean>((resolve) => {
+        handleSubmit(
+          (data) => {
+            onNext?.({ ...data });
+            resolve(true);
+          },
+          () => {
+            resolve(false);
+          }
+        )();
+      })
+  }) as unknown as FormProp<PaymentDataProps>);
+
+  return (
+    <MainContainer>
+      <FormContainer>
+        <div>
+          <Label>Forma de pago</Label>
+          <Select {...register("paymentPlanName")}>Plan de pago*
+            <option value="" disabled selected>Seleccione una opción</option>
```

Comment:
Este atributo no hace falta, de hecho en la consola tira un error diciendo que no debería estar


```suggestion
            <option value="" disabled>Seleccione una opción</option>
```